api/swagger.yml

```diff
@@ -1410,7 +1410,7 @@ components:
         state: pend
```

Comment:
Although it was not mentioned in the issue, the type `PostPoolL2Transaction` has the field `toBjj` https://github.com/hermeznetwork/hermez-node/blob/569150a6ccba235235a41348fa0aa03d583c1713/api/swagger.yml#L1189
This is an input type.  I think it makes sense to replace all instances of `Bjj` to `BJJ` in the API.  @AlbertoElias are you OK with more changes of `Bjj -> BJJ` than the one you reported?

Reply:
I'm fine. Though we do need to keep in mind that this is a breaking change unless you continue to support current naming until we move to a /v2

Reply:
Despite hermez-integration examples created by @Pantani and our frontend, are we aware of any other project is using our API so far? Because within project I believe we can manage this change.
However, because as we're live I don't think would be good to have a break changes and I would skip this change and create another issue to implement it in a new API version.

Reply:
I think this is small enough to be okay to wait for a new API version. But I do think there could be other changes that might need to be applied faster and would require some temporary backward compatible fixes.

Reply:
@AlbertoElias feel free to open another issue to describe these other changes you think might be necessaries.

Reply:
So, @ed255 I believe it would be better move forward this PR way it is and let this item you have pointed out in another task for another release.